src/main/java/taxi/model/Driver.java

```diff
@@ -6,13 +6,33 @@ public class Driver {
     private Long id;
     private String name;
     private String licenseNumber;
+    private String login;
+    private String password;
 
     public Driver() {
```

Comment:
I think we should override toString method here to show info about drivers in exception messages(but don't include password field)